package.json

```diff
@@ -60,6 +60,8 @@
       "prettier",
       "flowtype",
       "@emotion",
+      "import",
```

Comment:
I wonder - was it required for this PR? what problem does it solve here?

Reply:
It was needed for this override: https://github.com/emotion-js/emotion/blob/main/package.json#L99-L107

But [the docs for that rule state](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-commonjs.md) that it's meant to be temporary for transitioning to pure ES6 modules, so I think it's fine to remove this plugin and the override.